codalab/apps/teams/forms.py

```diff
@@ -89,3 +95,144 @@ def clean(self):
         cleaned_data = super(TeamMembershipForm, self).clean()
 
         return cleaned_data
+
+
+class OrganizerTeamForm(forms.ModelForm):
+
+    text_members = forms.CharField(required=False, widget=forms.Textarea)
+
+    class Meta:
+        model = Team
+        fields = ('description', 'name')
+
+    def __init__(self, *args, **kwargs):
+        self.user_list = []
+        self.competition = Competition.objects.get(pk=kwargs.pop('competition_pk'))
```

Comment:
Why do this here? instead of when cleaning/validating? seems like a weird place to do it?

Reply:
On `self.user_list`? PyCharm was specifying to include it in the constructor since it's something I reference later on. I previously had it in the specific method.

Reply:
Yeah, you should probably just do that all in a `clean()` method